src/static/js/ace2_inner.js

```diff
@@ -3374,7 +3374,7 @@ function Ace2Inner(){
       {
         try
         {
-          var newWindow = window.open(n.href, '_blank');
+          var newWindow = window.open(n.href, '_blank', 'noopener, noreferrer');
           newWindow.focus();
```

Comment:
this will probaby fail because `newWindow` will be null.

Could you verify it please? You should see an error message in the browser's console.